src/neo4j_genai/filters.py

```diff
@@ -90,7 +90,7 @@ class GteOperator(Operator):
 class InOperator(Operator):
     CYPHER_OPERATOR = "IN"
 
-    def cleaned_value(self, value):
+    def cleaned_value(self, value: str) -> str:
```

Comment:
`value` should be `list[float | int | str]` here.

Reply:
c9081d159d46cdaddbf90285271128c4cf7dc91a